specification/network/resource-manager/Microsoft.Network/stable/2024-01-01/examples/NetworkWatcherFlowLogGet.json

```diff
@@ -17,6 +17,7 @@
           "targetResourceId": "/subscriptions/subid/resourceGroups/rg/providers/Microsoft.Network/networkSecurityGroups/vm5-nsg",
           "targetResourceGuid": "00000000-0000-0000-0000-000000000000",
           "storageId": "/subscriptions/subid/resourceGroups/rg/providers/Microsoft.Storage/storageAccounts/wzstorage002",
+          "enabledFilteringCriteria": "srcIP=158.255.7.8 || dstPort=56891",
```

Comment:
>"enabledFilteringCriteria [](http://example.com/codeflow?start=10&length=25)

Why is this needed on a point GET request on a single resource?

Reply:
customer may want to know what filtering conditions have been set for a flowlogs resource. So, we return this information too in the GET call for flowlogs resource.